inst/examples/app/analytics/R/prepare-admin-panel.R

```diff
@@ -160,9 +53,14 @@ prepare_admin_panel_components <- function(
   })
 
   session_details <- shiny::reactive({
-    data_storage$read_session_data(
-      as.Date(input$date_from), as.Date(input$date_to)
-    )
+    tryCatch({
+      data_storage$read_session_data(
+        as.Date(input$date_from), as.Date(input$date_to)
+      )
+    },
+    error = function(e) {
+      return(NULL)
+    })
   })
```

Comment:
I had to this in case of session log files not available. As I can see, the example app does not log session files. Please correct me if I am wrong or if there is a better implementation.